diagrams/toolbox/generate-diagrams.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
+######################################################################
+#
+# Run the Cloudnet TOSCA toolbox.
+#
+# Copyright (c) 2021 Inria
+#
+######################################################################
+
+# Check that the CLOUDNET_BINDIR environment variable is set correctly.
+if [ ! -f ${CLOUDNET_BINDIR}/cloudnet_rc.sh ]
+then
+  echo "The CLOUDNET_BINDIR environment variable is not set correctly!"
+  exit 1
+fi
+
+# Load cloudnet commands.
+. ${CLOUDNET_BINDIR}/cloudnet_rc.sh
+
+# Copy the toolbox configuration.
+cp diagrams/toolbox/tosca2cloudnet.yaml .
+
+# Compile all xopera-examples service template files.
+translate $(grep 'tosca_definitions_version: ' -l -r .)
+
+# Remove useless generated files.
+rm diagrams/*-workflow-diagram.plantuml
+
+# Generate UML2 diagrams.
+generate_uml2_diagrams diagrams/*.plantuml
```

Comment:
I'd also rather see that each diagram would be separated the folder that contains the corresponding example.